htdocs/expedition/class/expedition.class.php

```diff
@@ -238,7 +245,7 @@ function create($user)
 		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
 		$sql.= ")";
 
-		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
```

Comment:
Yes documentation will improve quality of our work and we will be more 
efficient.
Do you really read the whole dev mailing list ? and keep in mind all 
remarks ??

I do not !

Le 02/04/2015 10:14, Marcos García a écrit :
>
> In htdocs/expedition/class/expedition.class.php 
> <https://github.com/Dolibarr/dolibarr/pull/2513#discussion_r27639112>:
>
> > @@ -238,7 +245,7 @@ function create($user)
> >  		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
> >  		$sql.= ")";
> >
> > -		dol_syslog(get_class($this)."::create", LOG_DEBUG);
> > +		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
>
> Well I think you should as you are making a PR.
>
> In fact, it was announced in the dev list 
> (http://lists.nongnu.org/archive/html/dolibarr-dev/2014-06/msg00001.html) 
> but documentation is obviously something that we must take care of.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Dolibarr/dolibarr/pull/2513/files#r27639112>.
>